src/Engine/Systems/PhysicsSystem.hpp

```
@@ -89,6 +94,12 @@ class PhysicsSystem {
 	double m_currentEpoch = 0.0;		// Current epoch (seconds past J2000) in ET
 	double m_simulationTime = 0.0;		// Simulation time (a.k.a. RELATIVE seconds elapsed since epoch; ABSOLUTE seconds is `epoch + m_simulationTime`)
 
+	struct _OrbitTrajectory {
```

Clang (14.0.6)

[warning] declaration uses identifier '_OrbitTrajectory', which is a reserved identifier

(bugprone-reserved-identifier)